glean-core/src/ping/mod.rs

```diff
@@ -40,7 +41,30 @@ impl PingMaker {
     }
 
     fn get_ping_seq(&self, glean: &Glean, storage_name: &str) -> usize {
-        1
+        // A internal category, not to be touched by anything else
+        let internal_storage = "glean.internal";
```

Comment:
We should be careful about this: we currently export some metrics in [`glean.internal.metrics`](https://github.com/mozilla/glean/blob/efdcd37604b4a6244478f406f5d5d9ad80604eff/glean-core/android/metrics.yaml#L42). I would do a few things:

- consider adding the Glean owned storage names to an internal document [like this](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/internal.md). If @mdboom has ported that over, we should maybe add this to that doc (was it #113 ?).
- check with @mdboom if we're forbidding `glean.internal` in glean_parser.

Reply:
`glean.internal.metrics` is a special category that the `glean_parser` only lets Glean itself use.  (Glean's internal build passes a special commandline flag to `glean_parser`).

Maybe we could just document this like the other internal metrics by adding it to `metrics.yaml` and set `send_in_pings` to `[]` so it will never be sent by accident on any pings?

Reply:
I'd like this, but then we'd be advertising a pattern (i.e. `send_in_pings: []`) which should be only used by us. I'm not sure of the top off my head if that has implications on the storage, too. @badboy , thoughts?